longest_common_subsequence/longestCommonSubsequence.java

```diff
@@ -0,0 +1,37 @@
+class LongestCommonSubsequence{
```

Comment:
Make this class public. Make the filename same as class name. Add space before `{`